src/routes/(console)/project-[region]-[project]/overview/+page.ts

```diff
@@ -5,9 +5,18 @@ import type { PageLoad } from './$types';
 import { selectedTab } from './store';
 
 export const load: PageLoad = async ({ params }) => {
-    if (get(selectedTab) === 'keys') {
-        redirect(302, `${base}/project-${params.region}-${params.project}/overview/keys`);
-    } else {
-        redirect(302, `${base}/project-${params.region}-${params.project}/overview/platforms`);
+    let subroute: string;
+    const tab = get(selectedTab);
+
+    switch (tab) {
+        case 'keys':
+        case 'dev-keys':
+            subroute = tab;
+            break;
+        default:
+            subroute = 'platforms';
+            break;
     }
+
+    redirect(302, `${base}/project-${params.region}-${params.project}/overview/${subroute}`);
```

Comment:
why not just this? we can save a lot of variables.

```suggestion
    const tab = get(selectedTab);

    switch (tab) {
        case 'keys':
        case 'dev-keys':
            redirect(302, `${base}/project-${params.region}-${params.project}/overview/${tab}`);
        default:
            redirect(302, `${base}/project-${params.region}-${params.project}/overview/platforms`);
    }
```

maybe also we shouldnt rely on the selectedTab variable to actually be the right path.